lib/Timber.php

```diff
@@ -196,24 +197,76 @@ public static function query_posts( $query = false, $PostClass = 'Timber\Post' )
 	/**
 	 * Get terms.
 	 * @api
-	 * @param string|array $args
-	 * @param array   $maybe_args
-	 * @param string  $TermClass
-	 * @return mixed
+	 * @param string|array $args a string or array identifying the taxonomy or
+	 * `WP_Term_Query` args. Numeric strings are treated as term IDs; non-numeric
+	 * strings are treated as taxonomy names. Numeric arrays are treated as a
+	 * list a of term identifiers; associative arrays are treated as args to
+	 * `WP_Term_Query::__construct()` and accepts any valid parameters to that
+	 * constructor.
+	 * @param array        $options optional; none are currently supported.
+	 * @return Iterable
+	 * @see https://developer.wordpress.org/reference/classes/wp_term_query/__construct/
+	 * @example
+	 * ```php
+	 * // Get all tags.
+	 * $tags = Timber::get_terms('post_tag');
+	 * // Note that this is equivalent to:
+	 * $tags = Timber::get_terms('post_tag');
+	 *
+	 * // Get all categories.
+	 * $cats = Timber::get_terms('category');
+	 *
+	 * // Get all terms in a custom taxonomy.
+	 * $cats = Timber::get_terms('my_taxonomy');
+	 *
+	 * // Perform a custom Term query.
+	 * $cats = Timber::get_terms([
+	 *   'taxonomy' => 'my_taxonomy',
+	 *   'orderby'  => 'slug',
+	 *   'order'    => 'DESC',
+	 * ]);
+	 * ```
 	 */
-	public static function get_terms( $args = null, $maybe_args = array(), $TermClass = 'Timber\Term' ) {
-		return TermGetter::get_terms($args, $maybe_args, $TermClass);
+	public static function get_terms( $args = null, array $options = [] ) : Iterable {
```

Comment:
For `$options`, we get a hint in #2208 as to what should be supported in the future: We could make the `merge` option work here. And we could even move the logic for that option from `Post::terms()` to this function.

Reply:
Great idea. #2211